test/benchmarks/Makefile

```diff
@@ -11,7 +11,7 @@ ifndef V
 endif
 
 $(TESTS):
-	$(QUIET_AGGREGATE)python3 ../aggregate.py $(TEST_ARGS) \
```

Comment:
Curious because I couldn't see the file benchmarks/test/Makefile in HEAD https://github.com/pytorch/xla. Where is the current file located?

Reply:
Oh I've added this PR on top of this other PR https://github.com/pytorch/xla/pull/6034 [still waiting for approval] to ease review. I was wondering why CI wasn't running, now I know why!
Will rebase onto master once the parent PR is merged. Will ping here once that happens.